packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -527,14 +527,15 @@ const SnowflakeDatasourceSpecificationEditor = observer(
           }
         />
         {/* TODO: we should reconsider adding this field, it's an optional boolean, should we default it to `undefined` when it's `false`?*/}
```

Comment:
Can you also remove this comment?

Reply:
I intentionally didn't remove this because I felt it's a valid point and maybe if in the future we have a better strategy we may want to use that here as well